api/RTCRtpSender.json

```diff
@@ -171,6 +173,231 @@
+            "support": {
+              "chrome": {
+                "version_added": "69"
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": "≤79"
+              },
+              "firefox": {
+                "version_added": "46",
+                "partial_implementation": true,
+                "notes": "The property is defined but not implemented/used."
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "11"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "return_object_property_degradationPreference": {
+          "__compat": {
+            "description": "<code>degradationPreference</code> property in returned object",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpSender/getParameters#degradationpreference",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "12.1"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": true
+            }
+          }
+        },
+        "return_object_property_encodings": {
+          "__compat": {
+            "description": "<code>encodings</code> property in returned object",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpSender/getParameters#encodings",
+            "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcrtpsendparameters-encodings",
+            "support": {
+              "chrome": {
+                "version_added": "69"
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": "≤79"
+              },
+              "firefox": {
+                "version_added": "46"
```

Comment:
Note, there was a note in RTCRtpSendParameters compat data that this was implemented in RTCRtpParameters.  However since we don't care about the dictionary I have simply put that this is same version as `getParameters()` introduced.